includes/admin/settings/class-settings-advanced.php

```diff
@@ -310,19 +310,30 @@ public function get_sections() {
 		 */
 		public function render_remove_cache_button( $field ) {
 			?>
-			<tr valign="top" <?php echo ! empty( $field['wrapper_class'] ) ? 'class="' . $field['wrapper_class'] . '"' : ''; ?>>
-				<th scope="row" class="titledesc">
-					<label
-						for="<?php echo esc_attr( $field['id'] ); ?>"><?php echo esc_html( $field['name'] ); ?></label>
-				</th>
-				<td class="give-forminp">
-					<button type="button" id="<?php echo esc_attr( $field['id'] ); ?>"
-							class="button button-secondary"><?php echo esc_html( $field['buttonTitle'] ); ?></button>
-					<?php echo Give_Admin_Settings::get_field_description( $field ); ?>
-				</td>
-			</tr>
-			<?php
-		}
+            <tr valign="top" <?php
+            echo !empty($field['wrapper_class']) ? 'class="' . $field['wrapper_class'] . '"' : ''; ?>>
+                <th scope="row" class="titledesc">
+                    <label
+                            for="<?php
+                            echo esc_attr($field['id']); ?>"><?php
+                        echo esc_html($field['name']); ?></label>
+                </th>
+                <td class="give-forminp">
+                    <button type="button" id="<?php
+                    echo esc_attr($field['id']); ?>"
```

Comment:
@jonwaldstein Is this file formatted correctly?

Reply:
resolved ab3d43c2